blocks/callout/src/app.tsx

```diff
@@ -24,7 +24,7 @@ export const App: BlockComponent<BlockEntityProperties> = ({
   const { graphService } = useGraphBlockService(blockRef);
   const { hookService } = useHookBlockService(blockRef);
 
-  useHook(hookService, editableRef, "text", "$.text", (node) => {
+  useHook(hookService, editableRef, "text", entityId, "$.text", (node) => {
```

Comment:
blocks are now required to pass the `entityId` of the entity they want to use the hook service for